src/utilities/ntrace/TraceConfiguration.cpp

```diff
@@ -129,8 +129,8 @@ void TraceCfgReader::readConfig()
 			PathName expandedName;
 
 			if (m_databaseName == noQuotePattern ||
-				expandDatabaseName(noQuotePattern, expandedName, nullptr) &&
-				m_databaseName == expandedName)
+				(expandDatabaseName(noQuotePattern, expandedName, nullptr),
+				m_databaseName == expandedName) )
```

Comment:
Why would you like to compare m_databaseName with garbage in expandedName if expandDatabaseName() failed?

Reply:
"false" returned by expandDatabaseName does not mean that we got garbage in expandedName. It just reports that alias was not found in databases.conf, rarely that info is (or 'was' - not sure for today) needed to the caller.

Reply:
expandedName always contains the most correct name of DB file